compiler+runtime/src/cpp/jank/read/parse.cpp

```diff
@@ -693,6 +695,53 @@ namespace jank::read::parse
     return object_source_info{ wrapped, start_token, sym_end };
   }
 
+  processor::object_result processor::parse_reader_macro_symbolic_values()
+  {
+    auto const start_token(token_current.latest.unwrap().expect_ok());
+    ++token_current;
+
+    auto sym_result(next());
+
+    if(sym_result.is_err())
+    {
+      return sym_result;
+    }
+    else if(sym_result.expect_ok().is_none())
+    {
+      return err(
+        error{ start_token.pos, native_persistent_string{ "value after ## must be present" } });
+    }
+    else if(sym_result.expect_ok().unwrap().ptr->type != object_type::symbol)
+    {
+      return err(
+        error{ start_token.pos, native_persistent_string{ "value after ## must be a symbol" } });
+    }
+
+    auto const sym(expect_object<obj::symbol>(sym_result.expect_ok().unwrap().ptr));
+    auto const sym_end(sym_result.expect_ok().unwrap().end);
+
+    native_real n;
```

Comment:
Always zero-initialize primitives. Otherwise, their values are undefined. Even if you're initializing it in every branch below now, if those branches change later, we'll be in for undefined behavior.

Reply:
Good point, I wasn't thinking about how the code could change later 👍